src/main/java/com/adobe/aem/compgenerator/utils/CommonUtils.java

```diff
@@ -251,4 +255,29 @@ private static BufferedWriter getFileWriterFromTemplate(String path,
         }
         return null;
     }
+    
+    
+    /**
+     * Gets the Tab properties if defined else returns all the properties.
+     *
+     * @param properties The {@link Property}
+     * @param tabs The {@link Tab}
+     * @return List<Property>
+     */
+    public static List<Property> getTabbedPropertiesIfExists(List<Property> properties, List<Tab> tabs) {
```

Comment:
I feel this function and it's javadoc are a bit misnamed.  This isn't getting tabbed properties as much as it is `sortPropertiesBasedOnTabs`, correct?  Or am I misunderstanding this code?

Reply:
Yeah, agreed updated the name.